src/test/java/org/folio/rest/support/db/ErrorFactory.java

```diff
@@ -0,0 +1,240 @@
+package org.folio.rest.support.db;
+
+import static org.folio.rest.support.db.ErrorConstants.CHECK_VIOLATION_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.DATATYPE_MISMATCH_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.DATA_LENGTH_MISMATCH_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.ERROR_TYPE;
+import static org.folio.rest.support.db.ErrorConstants.EXCLUSION_VIOLATION_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.FATAL_TYPE;
+import static org.folio.rest.support.db.ErrorConstants.FOREIGN_KEY_VIOLATION_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.INTEGRITY_VIOLATION__ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.INVALID_PASSWORD_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.INVALID_TEXT_REPRESENTATION_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.NOT_NULL_VIOLATION_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.PARENT_TABLE_NAME;
+import static org.folio.rest.support.db.ErrorConstants.UNIQUE_VIOLATION_ERROR_CODE;
+import static org.folio.rest.support.db.ErrorConstants.CHILD_TABLE_NAME;
+import static org.folio.rest.support.db.ErrorConstants.SCHEMA_NAME;
+
+import scala.collection.immutable.Map;
+
+public class ErrorFactory {
+
+  private ErrorFactory() {}
+
+ public  static Map<Object, String> getForeignKeyErrorMap(){
```

Comment:
Please put a space between the method parameters and the opening brace to be consistent with the code standards.

Other methods in this class are also affected, please correct those as well.

```suggestion
 public  static Map<Object, String> getForeignKeyErrorMap() {
```

Reply:
Done